gpii/node_modules/testing/src/CloudBased.js

```diff
@@ -74,6 +74,7 @@ gpii.test.cloudBased.gpiiConfig = function (baseDir) {
 };
 
 gpii.test.cloudBased.buildTestFixture = function (testDef, baseDir) {
+    testDef = fluid.copy(testDef);
```

Comment:
yeah - this is needed because the tests share the same testDef object, and the cloudBased.buildTestFixture modifies the object (screwing up any test sharing it)